src/fsharp/CompileOps.fs

```diff
@@ -3478,7 +3477,7 @@ let WriteOptimizationData (tcGlobals, file, ccu,modulInfo) =
 // Abstraction for project reference
 
 type RawFSharpAssemblyDataBackedByFileOnDisk (ilModule: ILModuleDef, ilAssemblyRefs) = 
-    let externalSigAndOptData = ["FSharp.Core";"FSharp.LanguageService.Compiler"]
```

Comment:
This removes the special treatment of "FSharp.LanguageService.COmpiler" where the sigdata and opdata were being placed into separate files.  There's no need to do this, it doesn't gain us much anyway (it was done to reduce DLL size when we were shipping the Visual F# Tools as a startup component in core Visual Studio, now that it's an optional install we don't need to do this any more, and in any case this only trimmed 1MB of DLL size, which isn't much in context)